plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/index.tsx

```diff
@@ -60,22 +61,22 @@ export const LYSPaymentsSteps: WooPaymentsProviderOnboardingStep[] = [
 		id: 'payment_methods',
 		order: 1,
 		type: 'backend',
-		label: 'Choose your payment methods',
+		label: __( 'Choose your payment methods', 'woocommerce' ),
 		content: <PaymentMethodsSelection />,
 	},
 	{
 		id: 'wpcom_connection',
 		order: 2,
 		type: 'backend',
-		label: 'Connect with WordPress.com',
+		label: __( 'Connect with WordPress.com', 'woocommerce' ),
```

Comment:
Use a placeholder for "WordPress.com" (i.e. "Connect with %s").

Reply:
Thanks, I agree that WordPress.com should not be translatable. Updated!